package org.celery.exceptions

class CommandAbortException: Throwable() {

}
